### PR TITLE
Added tests

### PR DESCRIPTION
**Pull request by jordisala1991**
not sure if it covers everything, but its something

### CODE REVIEW

File: Tests/Admin/BaseMediaAdminTest.php [hunk 1]
```diff
@@ -35,10 +46,57 @@ protected function setUp()
...
+        $context = $this->prophesize();
+        $context->willExtend('Sonata\MediaBundle\Tests\Admin\EntityWithGetId');
+        $context->willImplement('Sonata\ClassificationBundle\Model\ContextInterface');
+
+        $this->configureGetPersistentParameters();
+        $this->configureGetProviderName($media);
+        $this->modelManager->getModelInstance('Sonata\MediaBundle\Entity\BaseMedia')->willReturn($media->reveal());
+        $this->categoryManager->find(1)->willReturn($category->reveal());
+        $this->request->isMethod('POST')->willReturn(true);
+        $this->request->get('context')->willReturn($context->reveal());
```
Comment (jordisala1991) [inline Tests/Admin/BaseMediaAdminTest.php]: this is wrong, have to change it

File: Tests/Admin/BaseMediaAdminTest.php [hunk 2]
```diff
@@ -35,10 +46,57 @@ protected function setUp()
             $this->pool->reveal(),
             $this->categoryManager->reveal()
         );
+        $this->mediaAdmin->setRequest($this->request->reveal());
+        $this->mediaAdmin->setModelManager($this->modelManager->reveal());
+        $this->mediaAdmin->setUniqid('uniqid');
+    }
+
+    public function testGetNewIntance()
```
Comment (greg0ire) [inline Tests/Admin/BaseMediaAdminTest.php]: Instance

File: Tests/Admin/BaseMediaAdminTest.php [hunk 3]
```diff
@@ -35,10 +46,57 @@ protected function setUp()
...
+        $this->configureGetProviderName($media);
+        $this->modelManager->getModelInstance('Sonata\MediaBundle\Entity\BaseMedia')->willReturn($media->reveal());
+        $this->categoryManager->find(1)->willReturn($category->reveal());
+        $this->request->isMethod('POST')->willReturn(true);
+        $category->getContext()->willReturn($context->reveal());
+        $media->setContext('context')->shouldBeCalled();
+
+        $new_media = $this->mediaAdmin->getNewInstance();
+
+        $this->assertSame($media->reveal(), $new_media);
```
Comment (greg0ire) [inline Tests/Admin/BaseMediaAdminTest.php]: use camel case for `$newMedia` please

Comment (greg0ire) [inline Tests/Admin/BaseMediaAdminTest.php]: Or don't even use this variable, use the function call directly inside `assertSame`

Comment (jordisala1991) [inline Tests/Admin/BaseMediaAdminTest.php]: Not sure about calling the method inside the assertSame, but I changed it

**Comment by enekochan** [comment]
Tests passed and works OK. Thank you!

**Pull request merged**